core/arch/arm/plat-imx/imx6ul.c

```diff
@@ -32,4 +33,5 @@ static void init_csu(void)
 void plat_cpu_reset_late(void)
 {
 	init_csu();
+	init_caam();
```

Comment:
Any reason why this is done in `plat_cpu_reset_late`? At least for this case the earliest user is the linux driver, in that case it can be moved to normal platform init.

Reply:
No reason at all. I located it with csu_init() because it _seemed_ like the right place.
I can move it elsewhere - I don't think there's any barrier to that.

Reply:
`plat_cpu_reset_late` happens before the console is initialized, but since your code does not print anyway this is a minor nit on my part. If you go the route of implementing this into the driver subdir, this may be the correct place afterall, since the full CAAM implementation (if I remember correctly) should be initialized in `plat_cpu_reset_late` anyway. Take this with a grain of salt, I have not looked deep ito the code